DigitalLearningSolutions.Web/Views/TrackingSystem/Centre/Reports/_ActivityTable.cshtml

```diff
@@ -19,30 +25,26 @@
     </tr>
   </thead>
   <tbody class="nhsuk-table__body">
-    @{
-      var rowCount = 0;
-      foreach (var activityRow in Model.Rows)
-      {
-        rowCount++;
-        <tr role="row" class="nhsuk-table__row @(rowCount > 5 ? "js-hidden-activity-row" : string.Empty)">
-          <td role="cell" class="nhsuk-table__cell">
-            <span class="nhsuk-table-responsive__heading">Period </span>@activityRow.Period
-          </td>
-          <td role="cell" class="nhsuk-table__cell">
-            <span class="nhsuk-table-responsive__heading">Registrations </span>@activityRow.Registrations
-          </td>
-          <td role="cell" class="nhsuk-table__cell">
-            <span class="nhsuk-table-responsive__heading">Completions </span>@activityRow.Completions
-          </td>
-          <td role="cell" class="nhsuk-table__cell">
-            <span class="nhsuk-table-responsive__heading">Evaluations </span>@activityRow.Evaluations
-          </td>
-        </tr>
-      }
+    @foreach (var (activityRow, rowNumber) in Model.Rows.Select((row, rowNumber) => (row, rowNumber)))

```

Comment:
When thinking about rows in a table, I think most people would more naturally think of the first row as corresponding to "row number = 1" rather than "row number = 0", so I think it's worth changing that.